@xen-orchestra/xapi/host.mjs

```diff
@@ -162,6 +162,33 @@ class Host {
 
     return ipmiSensorsByDataType
   }
+
+  async getBiosInfo(ref, { cache }) {
+    const biosData = await this.call('host.get_bios_strings', ref)
+    const { 'bios-version': currentBiosVersion, 'system-product-name': hostServerName } = biosData
+
+    if (!cache.has('servers')) {
+      const response = await fetch(
+        'https://pictures.2cr.si/Images_site_web_Odoo/Pages_produit/VATES-BIOS_BMC_last-version.json'
```

Comment:
I'm really not fond calling this URL without testing first that the host might be a 2CRSI machine.

Any way we can improve it?

Reply:
We could add a verification on the ```system-manufacturer ``` value containing ```2CRSI``` before fetching the URL?

Reply:
It looks promising but I have no knowledge on how it works, I let you discuss this with the other reviewers :slightly_smiling_face: